curriculum/challenges/english/24-b1-english-for-developers/learn-about-adverbial-phrases/6807f237eafb11ce7bbc1b01.md

```diff
@@ -0,0 +1,80 @@
+---
+id: 6807f237eafb11ce7bbc1b01
+title: Task 48
+challengeType: 22
+dashedName: task-48
+---
+
+<!-- (Audio) Sophie: Right now, we're using our central servers. -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Right now, we're using our central BLANK.`
+
+## --blanks--
+
+`servers`
+
+### --feedback--
+
+This refers to computers or systems that provide services or resources to other computers or devices on a network.
```

Comment:
```suggestion
This word in the plural form refers to computers or systems that provide services or resources to other computers or devices on a network.
```

Reply:
Again, I take it this is due to the way we've been structuring the feedbacks, but right after I mention 'computers' and 'systems' in the plural. I'm adding those to stick to the structure.